<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ComplectProductWindow</class>
 <widget class="QDialog" name="ComplectProductWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>865</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Установка оборудования</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>10</y>
     <width>341</width>
     <height>501</height>
    </rect>
   </property>
   <property name="title">
    <string>Свободные модули</string>
   </property>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>60</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>поиск по серийному №</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leNumModSearch">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>80</y>
      <width>211</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="tbSearchModul">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>80</y>
      <width>23</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QListWidget" name="lwOuterModule">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>321</width>
      <height>371</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbNewModule">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Новые</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbOldModule">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Отремонтировнные</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pbAddModul">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>240</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbDeleteModul">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>270</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;&gt;</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>371</width>
     <height>501</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>false</bold>
    </font>
   </property>
   <property name="title">
    <string>Изделие</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Модули в составе</string>
    </property>
   </widget>
   <widget class="QListWidget" name="lwInnerModule">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>341</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>90</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Наименование</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbNameProd">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>110</y>
      <width>240</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="leNumProdSearch">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>51</y>
      <width>201</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="tbProdSearch">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>50</y>
      <width>23</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>31</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Поиск по серийному №</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>90</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Серийный №</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbNumProd">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>90</y>
      <width>240</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pbOK">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>520</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCancel">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>520</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Отменить</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>leNumProdSearch</tabstop>
  <tabstop>leNumModSearch</tabstop>
  <tabstop>rbNewModule</tabstop>
  <tabstop>rbOldModule</tabstop>
  <tabstop>pbAddModul</tabstop>
  <tabstop>pbDeleteModul</tabstop>
  <tabstop>pbOK</tabstop>
  <tabstop>pbCancel</tabstop>
  <tabstop>lwOuterModule</tabstop>
  <tabstop>lwInnerModule</tabstop>
  <tabstop>tbSearchModul</tabstop>
  <tabstop>tbProdSearch</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pbCancel</sender>
   <signal>clicked()</signal>
   <receiver>ComplectProductWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>469</x>
     <y>531</y>
    </hint>
    <hint type="destinationlabel">
     <x>432</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
